.travis.yml

```diff
@@ -25,7 +25,7 @@ before_install:
     - wget -O - http://llvm.org/apt/llvm-snapshot.gpg.key | sudo apt-key add -
```

Comment:
Little late... but would it be worth switching this to curl too for consistency?
(e.g. `curl http://llvm.org/apt/llvm-snapshot.gpg.key | sudo apt-key add -`)